test/puppetlabs/puppetdb/testutils/jetty.clj

```diff
@@ -44,22 +74,31 @@
 
 (defn puppetdb-instance
   "Stands up a puppetdb instance with `config`, tears down once `f` returns.
-   Adjusts *server* and *base-url* to refer to the instance."
+  If the port is assigned by Jetty, use *port* to get the currently running port."
   ([f] (puppetdb-instance (create-config) f))
-  ([config f]
-   (let [config (conf/adjust-tk-config config)
+  ([config f] (puppetdb-instance config [] f))
+  ([config services f]
```

Comment:
Is this new services arg for future use? (And if so, wonder if we should mention it in the docstring and/or commit message.)